hugegraph-dist/src/assembly/static/scripts/example.groovy

```diff
@@ -15,53 +15,55 @@
  * under the License.
  */
 import org.apache.hugegraph.HugeFactory
+import org.apache.hugegraph.backend.id.IdGenerator
 import org.apache.hugegraph.dist.RegisterUtil
+import org.apache.hugegraph.type.define.NodeRole
```

Comment:
tiny question: I didn't import this line before and it seems to work too, weird

Reply:
I guess that some classes have been imported after `plugin activated: HugeGraph`.

```bash
> ./bin/gremlin-console.sh

         \,,,/
         (o o)
-----oOOo-(3)-oOOo-----
plugin activated: HugeGraph
plugin activated: tinkerpop.server
plugin activated: tinkerpop.utilities
plugin activated: tinkerpop.tinkergraph
gremlin> NodeRole
==>class org.apache.hugegraph.type.define.NodeRole
```